prusti-viper/src/encoder/spec_encoder.rs

```diff
@@ -171,6 +172,111 @@ impl<'p, 'v: 'p, 'tcx: 'v> SpecEncoder<'p, 'v, 'tcx> {
                     .collect(),
                 self.encode_assertion(body),
             ),
+            box typed::AssertionKind::SpecEnt(ref cl, ref vars, ref pres, ref posts) => {
```

Comment:
It would be better to move the implementation of this case to a separate function, to avoid reaching 12 levels of indentation.